benchopt/runner.py

```diff
@@ -396,5 +396,5 @@ def run_benchmark(benchmark, solver_names=None, forced_solvers=None,
 
     if plot_result:
         from benchopt.plotting import plot_benchmark
-        plot_benchmark(df, benchmark)
+        plot_benchmark(df, benchmark, save_file)
```

Comment:
Overall, I would rather not have to pass `save_file` here. But if it is too complicated, let it be :)

Reply:
It is the main cause of tests failures, I'm trying to remove it but it seems complicated... However, adding it in the other files is not always possible so we will need to discuss about that in more details ^^